PoS - node deploy guides update (#166)
0xPolygon/polygon-docs: docs/pos/how-to/snapshots.md

When setting up a new sentry, validator, or full node server, it is recommended that you use snapshots for faster syncing without having to sync over the network. Using snapshots will save you several days for both Heimdall and Bor. 

!!! note

    We no longer support Bor archive snapshots due to unsustainable data growth.

!!!tip
    
    For the latest snapshot, please visit [<ins>Polygon Chains Snapshots</ins>](https://snapshot.polygon.technology/).

## Client snapshots

> To begin, ensure that your node environment meets the **prerequisites** outlined [here](https://docs.polygon.technology/pos/operate-node/operate/full-node-binaries/). Before starting any services, execute the shell script provided below. This script will download and extract the snapshot data, which allows for faster bootstrapping. In our example, we will be using an Ubuntu Linux m5d.4xlarge machine with an 8TB block device attached.

Reviewer: ```suggestion
To begin, ensure that your node environment meets the **prerequisites** outlined [here](https://docs.polygon.technology/pos/operate-node/operate/full-node-binaries/). Before starting any services, execute the shell script provided below. This script downloads and extracts the snapshot data, which allows for faster bootstrapping. In our example, we use an Ubuntu Linux m5d.4xlarge machine with an 8TB block device attached.
```

Reviewer: prefer present simple over future with will

| approx. compressed total | 210 GB (erigon) + 35 GB (heimdall) | 245 GB |
| approx. data growth daily | 4.5 GB (erigon) + .5 GB (heimdall) | 5 GB |
| approx. total extracted size | 875 GB (erigon) + 50 GB (heimdall) | 925 GB |
| suggested disk size (2.5x buffer) | 925 GB * 2.5 (natural chain growth) | 2.5 TB | 

**Note:** 

> PoS Network is deprecating Archive Node snapshots we request users to move to the Erigon Client and make use of Erigon Snapshots.

Reviewer: could use an admonition here instead, this note format is unusual ..

!!! note
    note here
